swift_kernel.py

```diff
@@ -187,16 +188,56 @@ def _init_swift(self):
         This must happen after package installation, because the ClangImporter
         does not see modulemap files that appear after it has started."""
 
+        self.send_response(self.iopub_socket, 'stream', {
```

Comment:
Remove these messages from `_init_swift()` because they'll appear if you execute non-installation code in the first cell, and I think that they shouldn't.

Of course, we'll lose these status messages during installation, but I don't think they're very important.